public/bitemp_graph_helper.js

```diff
@@ -289,7 +288,7 @@ function saveNewDoc(chart) {
     data: data,
     processData: false,
     success: function(data) {
-      if(!window.location.href.endsWith('/search')) {
+      if(!window.location.href === 'http://localhost:3000/') {
```

Comment:
Don't rely on the server being localhost or the port being 3000 -- what if you're accessing the app running on a different server? what if it's deployed to a different port?